ocaml/xapi/cli_operations.ml

```diff
@@ -208,14 +208,19 @@ let host_license_of_r host_r editions =
     else
       0.
   in
-  let edition = host_r.API.host_edition in
-  let edition_short = List.hd
-      (List.filter_map (fun (a, (_, b, _)) -> if a = edition then Some b else None) editions) in
+  let edition_str = host_r.API.host_edition in
+  let edition_short =
+    editions
+    |> List.filter_map
+     V6_interface.(fun ed -> if ed.ed_title = edition_str
+                                then Some ed.ed_code
+                                else None)
+    |> List.hd in
   {
     hostname = host_r.API.host_hostname;
     uuid = host_r.API.host_uuid;
     rstr = rstr;
-    edition = edition;
+    edition = edition_str;
```

Comment:
What's the benefit of the rename?

Reply:
I think I did that to keep some consistency with the renaming in v6d, but you're right in that it serves no purpose here so I'll remove it.

Reply:
Satisfying the type system, which will otherwise recognise `edition` as a generated RPC type.

Reply:
Another way to deal with this would be to change the type name from edition to edition_repr or edition_rec, edition_type, or something